### ISSUE TITLE
Better Documentation for Python(3)

### ISSUE THREAD

**Issue by lapplislazuli** [antlr/grammars-v4#2358]
Hi, 

I am just getting started with Antlr, and I want to try some AST Manipulation for Python. 
Honestly I am having a hard time, but I think everything is working I am just using it wrong. 

For now I came from the [getting started](https://github.com/antlr/antlr4/blob/master/doc/getting-started.md) and wanted to parse a python3 file to get a similar tree shown. 

So there are a few things when starting from the toplevel repository that I encountered: 

- When I go to the Python-Folder, I see many different versions. It would be nice to get a short statement how I choose the right grammar/subfolder. Particularly the existence of Python, Python2 and Python3 is a bit confusing. 
- "Just running" Antlr worked, I did that for both Lexer and Parser, but the `javac Python*.java` did not work as the `Python3LexerBase.java` was not found. I had to copy it upwards, but ... I think that is not how this should be solved. 
- After this, compilations worked fine 
- Using `grun Python3Parser -gui sample.py` fails with 
```
Exception in thread "main" java.lang.ClassCastException: class Python3Parser
	at java.base/java.lang.Class.asSubclass(Class.java:3851)
	at org.antlr.v4.gui.TestRig.process(TestRig.java:135)
	at org.antlr.v4.gui.TestRig.main(TestRig.java:119)
```
  I think I need some toplevel parser class too? Or I have to use it in a certain context? 
- Using `grun Python3Lexer -gui sample.py` does fail as there is no Python3LexerParser but that seems reasonable
- From the Wiki I found [This example project](https://github.com/teverett/antlr4example) but it is a full maven program. Does that imply that I cannot/should not use the commandline? 

To sum it up: 
As someone new to Antlr, coming from the getting started is a bit confusing for three reasons
1. The Getting-Started approach does not work
2. I have a hard time selecting the right grammar and I am rather guessing which one is good 
3. The example from the wiki is very specific for Maven and does not give me explanations what is happening
